### PR TITLE
Changelog for 1.4.0-beta

### PR DESCRIPTION
**Pull request by sergeybykov**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -7,6 +7,58 @@ The idea is to track end-user facing changes as they occur.*
...
+  - In XML config, if SystemStoreType set to Custom but no ReminderTableAssembly are specified, assume that ReminderServiceProviderType is set to Disabled #2589
+  - In config XML, when SystemStoreType is set to MembershipTableGrain, set ReminderServiceType to ReminderTableGrain #2590
+  - Service Fabric cluster membership providers #2542
+  - Adds optional native JSON support to MySQL #2288
+  - Azure table storage throws InconsistentStateException #2630
+  - Allow serializers to have multiple [Serializer(...)] attributes #2611
+  - Removed GrainStateStorageBridge from GrainCreator to allow better control of the IStorage used when using non-silo unit tests. #2243
+  - Failsafe Exception serialization #2633
+  - Added a data adapter to azure queue stream provider #2658
+  Client cluster disconnection #2628
```
Comment (ReubenBond) [inline changelog.md]: missing  a `-`?

Comment (sergeybykov) [inline changelog.md]: I just copy-pasted the original commit message. :-) You are right, "client-cluster" seems better here.

Comment (ReubenBond) [inline changelog.md]: I meant before the line, so it shows up as a dot-point. Maybe I'm wrong, though

Comment (sergeybykov) [inline changelog.md]: Oh, yes, of course.

**Pull request merged**